images/docker-compose-mca.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  ioc:
+    image: ghcr.io/cnpem/mca-epics-ioc:$TAG
+    build:
+      context: ./
+      dockerfile: ../Dockerfile
+      target: no-build
```

Comment:
Same thing here, I think we should support static linking the IOCs... (though they would probably remain `no-build` here)

It doesn't have to be changed right now, though.

Reply:
It was not possible to update to "static-link", a failure is generated in the Dockerfile when copying the RELEASE file.